src/gpu/AssemblerAlign-gpu.cpp

```diff
@@ -124,8 +124,8 @@ void Assembler::computeAlignmentsGpu(
     cout << timestamp << "Creating alignment table." << endl;
     computeAlignmentTable();
 
-    cout << timestamp << "Shutting down processors." << endl;
-    shasta_shutdownProcessors(nDevices);
+    //cout << timestamp << "Shutting down processors." << endl;
```

Comment:
Fine to leave the message commented out, but nevertheless I would make it less ambiguous by changing "processors" to "GPUs" or something else. Most people would normally interpret "processors" as meaning "CPUs".